docs/source/services/timelock_service/paxos.rst

```diff
@@ -0,0 +1,40 @@
+.. _timelock-paxos:
+
+=====
+Paxos
+=====
+
+Paxos is an algorithm for *distributed consensus* - that is, getting servers to agree on a value that some server
+*proposes*, in the presence of communication and server failures. Its guarantees include:
+
+#. Safety: All servers will agree on the same value.
+#. Nontriviality: The value must have been *proposed* by at least one of the servers.
+#. Liveness: If a majority of servers is up and able to communicate, then the protocol can make progress.
+
+If the Timelock Server is configured to use Paxos, it will actually use separate instances of Paxos to agree on the
```

Comment:
Ah, okay. Yeah we don't officially support anything else at the moment.

Yep, that describes what I was going for here (the more interesting part is how we use Paxos, really). I'll reorganise this to fit that outline.